app/backend/templates/host_reference.md

```diff
@@ -15,7 +15,9 @@ This is what they wrote:
 
 Thanks for using Couchers to organize this interaction! We hope you had an enjoyable, fulfilling time.
 {% else %}
-Please go and write a reference for them too. It's a nice gesture and helps us build a community together! When you've both written a reference, both references will become visible. Otherwise {{ reference.from_user.name|couchers_escape }}'s reference will become visible 2 weeks after the end of your interaction, after which you cannot write a reference back.
+Please go and write a reference for them too.
+{{ urls.leave_reference_link | couchers_escape}}
+ It's a nice gesture and helps us build a community together! When you've both written a reference, both references will become visible. Otherwise {{ reference.from_user.name|couchers_escape }}'s reference will become visible 2 weeks after the end of your interaction, after which you cannot write a reference back.
```

Comment:
Please format this link similar to other emails with a button on HTML, and link otherwise